rust/arrow/src/compute/kernels/boolean.rs

```diff
@@ -111,7 +111,22 @@ pub fn not(left: &BooleanArray) -> Result<BooleanArray> {
     Ok(BooleanArray::from(Arc::new(data)))
 }
 
-pub fn is_null(input: &ArrayRef) -> Result<BooleanArray> {
+/// Returns a non-null [BooleanArray] with whether each value of the array is null.
+/// # Error
+/// This function never errors.
+/// # Example
+/// ```rust
```

Comment:
What I do in doctests and comments is give a space between each of them like:
```suggestion
/// # Error
///
/// This function never errors.
/// # Example
///
/// ```rust
```